evm_arithmetization/src/lib.rs

```diff
@@ -217,7 +217,7 @@ pub mod util;
 use jemallocator::Jemalloc;
 use mpt_trie::partial_trie::HashedPartialTrie;
 
-#[cfg(not(target_env = "msvc"))]
+#[cfg(not(any(target_env = "msvc", jemalloc_disable)))]
```

Comment:
I would eventually add a TODO linking the related issue. We don't know "yet" whether jemalloc brings a non-negligible performance boost or not. If it does, then I'm not sure we'd want to stick with this option in the long term.

Reply:
Agreed. I'll add this.